pkg/azureconstants/azure_constants.go

```diff
@@ -111,6 +111,7 @@ const (
 
 var (
 	// ManagedDiskPath is described here: https://docs.microsoft.com/en-us/rest/api/compute/disks/createorupdate#create-a-managed-disk-from-an-existing-managed-disk-in-the-same-or-different-subscription.
-	ManagedDiskPath   = "/subscriptions/%s/resourceGroups/%s/providers/Microsoft.Compute/disks/%s"
-	ManagedDiskPathRE = regexp.MustCompile(`(?i).*/subscriptions/(?:.*)/resourceGroups/(?:.*)/providers/Microsoft.Compute/disks/(.+)`)
+	ManagedDiskPath    = "/subscriptions/%s/resourceGroups/%s/providers/Microsoft.Compute/disks/%s"
+	ManagedDiskPathRE  = regexp.MustCompile(`(?i).*/subscriptions/(?:.*)/resourceGroups/(?:.*)/providers/Microsoft.Compute/disks/(.+)`)
```

Comment:
what is the change to make it case insensitive?

Reply:
The constants file already has the case insensitive regex; however, [azure_controller_common.go](https://github.com/kubernetes-sigs/azuredisk-csi-driver/pull/3149/files#diff-e8f8f37f77f8a5146c12a9dfdeffa6e498457da7ad360d259061160c29ab9e64) was redefining the same regex without the case insensitive part. I've changed all the references to use the correct regex defined in the constants.